### PR TITLE
chore: Adding release-please annotations to readme files

### PR DESCRIPTION
**Pull request by ddixit14**

### CODE REVIEW

File: synthtool/gcp/templates/java_library/README.md [hunk 1]
```diff
@@ -37,6 +37,7 @@ If you are using Maven with [BOM][libraries-bom], add this to your pom.xml file:
 If you are using Maven without BOM, add this to your dependencies:
 {% else %}
 If you are using Maven, add this to your pom.xml file:
+<!--- {x-version-update-start:{{ artifact_id }}:released} -->
```
Comment (suztomo) [inline synthtool/gcp/templates/java_library/README.md:40]:
```suggestion
<!-- {x-version-update-start:{{ artifact_id }}:released} -->
```

File: synthtool/gcp/templates/java_library/README.md [hunk 2]
```diff
@@ -72,6 +73,7 @@ If you are using SBT, add this to your dependencies:
 ```Scala
 libraryDependencies += "{{ group_id }}" % "{{ artifact_id }}" % "{{ metadata['latest_version'] }}"
 ```
+<!--- {x-version-update-end} -->
```
Comment (suztomo) [inline synthtool/gcp/templates/java_library/README.md:76]:
```suggestion
<!-- {x-version-update-end} -->
```

**Review by suztomo — APPROVED** [review]
Remove unnecessary "-" in comment.

File: synthtool/gcp/templates/java_library/README.md (hunk 2 above)
Comment (ddixit14) [reply synthtool/gcp/templates/java_library/README.md]: Done

File: synthtool/gcp/templates/java_library/README.md (hunk 1 above)
Comment (ddixit14) [reply synthtool/gcp/templates/java_library/README.md]: Done

**Review by diegomarquezp — APPROVED** [review]

**Pull request merged**